redfish-core/include/registries/privilege_registry.hpp

```diff
@@ -543,6 +543,9 @@ const static auto& putLogEntry = privilegeSetConfigureManager;
 const static auto& deleteLogEntry = privilegeSetConfigureManager;
 const static auto& postLogEntry = privilegeSetConfigureManager;
 
+// CELogEntry
+const static auto& getCELogEntry = privilegeSetConfigureManager;
```

Comment:
It would probably just be best if you did this in line, since at the top this says do not edit, generated.

Reply:
Makes sense. Will revert the changes.